components/Login/Login.js

```diff
@@ -0,0 +1,62 @@
+import React from "react";
+import { View, Text, TextInput, Button, Image } from "react-native";
+import { useState } from "react";
+import styles from './LoginStyles'
+
+function Login(){
+
+    const [ name, setName ] = useState('');
+    const [ password, setPassword ] = useState('');
+
+    function handleLogin(name,password){
+        if(!name || !password){
+            return;
+        }
+        else{
+            
+        }
+    }
+
+    return (
+        <View style={styles.container}>
+            <View style={styles.logoContainer}>
+                <Image
+                    source={require('../../assets/tackleboxlogo.png')}
+                    style={styles.logoImage}
+                    resizeMode="contain"
+                    />
+            </View>
+            <View style={styles.loginContainer}>
+                <TextInput
+                    testID='userName-input'
+                    maxLength={12}
+                    style={styles.input}
+                    placeholder="Username"
+                    value={name}
+                    onChangeText={setName}
+                />
+                <TextInput
+                    testID='password-input'
+                    maxLength={16}
+                    style={styles.input}
+                    placeholder="Password"
+                    value={password}
+                    onChangeText={setPassword}
+                    secureTextEntry={true}
```

Comment:
Nice UX consideration! That'll set us up for the "View Password" eyeball feature